### PR TITLE
Fixes two issues with hideable column toggle components

### LINKED ISSUE #1694: Fix datagrid hideable column toggle component
**Select one ...**  (check one with "x")
```
[ x ] bug
[ ] feature request
[ ] enhancement
```

### Expected behavior

It should have the correct layout and active state

### Actual behavior
Datagrid hideable column toggle component has incorrect layout and active state
- When selection is enabled on the data grid its positioned in the middle of the footer
- the active state is incorrect

### Reproduction of behavior
screen shot 2017-11-15 at 9 47 51 am
screen shot 2017-11-15 at 9 48 50 am

### Environment details

* **Angular version:** 4.x.x

* **Clarity version: @latest

* **OS and version:** 

* **Browser:** [all]

### PR DESCRIPTION
**Pull request by hippee-lee**
- Fixes layout to align to left side of footer and expand to fill space between itself and the right side of the footer (or pagination if that component is there)
- Fixes active state
- closes #1694

## Before
screen shot 2017-11-15 at 9 47 51 am
screen shot 2017-11-15 at 9 48 50 am

## After
screen shot 2017-11-15 at 9 49 24 am
screen shot 2017-11-15 at 9 49 34 am

## Tested on
- Chrome
- Safari
- Firefox
- IE
- Edge

### CODE REVIEW

File: src/clarity-angular/data/datagrid/_datagrid.clarity.scss [hunk 1]
```diff
@@ -652,25 +648,23 @@
...
+                }
+            }
             .column-toggle--action {
-
                 // I'm overriding .btn/.btn-link here but am not confident this is the correct way to do it.
                 min-width: 0.75rem;
                 padding-left: 0;
                 padding-right: 0;
 
-                clr-icon {
```
Comment (adityarb88) [inline src/clarity-angular/data/datagrid/_datagrid.clarity.scss]: Why is this removed?

Comment (hippee-lee) [reply src/clarity-angular/data/datagrid/_datagrid.clarity.scss]: The component doesn't need to target the clr-icon element directly anymore. I just target the `.column-toggle--action` class and let it cascade down. I think I wrote it originally before I really understood how clr-icons worked wrt setting color.

Comment (adityarb88) [reply src/clarity-angular/data/datagrid/_datagrid.clarity.scss]: Awesome. Thanks!
